lib/protobuf/gen/state.pb.ex

```diff
@@ -199,6 +205,20 @@ defmodule ForgeAbi.ForgeState.StakeSummaryEntry do
   field :value, 2, type: ForgeAbi.StakeSummary
 end
 
+defmodule ForgeAbi.ForgeState.GasEntry do
```

Comment:
what is this? looks like you're using the map<> but the generated code still has this?

Reply:
Yes this is generated by the map<>.